simue/handler.go

```diff
@@ -480,6 +481,8 @@ func HandleProcedure(ue *simuectx.SimUe) {
 		ue.Log.Infoln("Initiating UE Triggered Service Request Procedure")
 		msg := &common.UeMessage{}
 		msg.Event = common.SERVICE_REQUEST_EVENT
+		ue.Log.Infoln("Waiting for 60 Secs. Restart amf if you want to test amf state store feature ...")
```

Comment:
OK. This is what we should do in separate code changes. Delay after every procedure. Instead of keeping 60 seconds ?

Reply:
Yes. this we can start taking config from profile and add delay